### ISSUE TITLE
Tensorboard

### ISSUE THREAD

**Issue by mahfuj9346449** [kratzert/finetune_alexnet_with_tensorflow#14]
I cannot obtain the test accuracy from tensor-board.  Can you provide any example of Embedding Visualization, please ?

**Comment by kratzert** [comment]
The test accuracy is one number at the end of your training? Why or how you want to visualize this in the training graph? Or do you mean the validation accuracy?
I don't have much time at the moment to guide you with code, but note that this code repository was meant for people with basic knowledge of TensorFlow. Should be to hard for you to add validation accuracy to the graph if you want.

I usually just observe the evolution of the training accuracy in the graph, print validation after each epoch to the shell and well test accuracy is a completely different story. This is calculate once after training by a separate script (since you don't want to finetune against the test accuracy!!) and therefore it's sufficient to print this one number to shell.

**Comment by mahfuj9346449** [comment]
Yes, I meant validation accuracy. Thanks.
